lib/routes/token.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var validators = require('./validators')
+var HEX_STRING = validators.HEX_STRING
+
+module.exports = function (log, isA, error, db) {
+
+  var routes = [
+    {
+      method: 'POST',
+      path: '/token/verify',
+      config: {
+        validate: {
+          payload: {
+            code: isA.string().min(32).max(32).regex(HEX_STRING).required()
+          }
+        }
+      },
+      handler: function (request, reply) {
+        log.begin('Token.verify', request)
+        // TODO - Do something real here
+        reply({})
+      }
+    },
+    {
+      method: 'GET',
+      path: '/token/status',
```

Comment:
I'm trying to think about how old clients will work, and whether we actually need a new endpoint if we can get by with what we have.

In the first question, I wrote the wrong endpoint, signup polling occurs on `/recovery_email/status`, which returns `verified: false` or `verified: true` depending on whether the email has been verified.